addroom.html

```diff
@@ -27,33 +26,46 @@
             /* vertical-align: middle; */
             /* background-clip:border-box; */
             height:auto;
+            width: 105.08%;
             line-height: auto;
             background-color:transparent;
 
       }
       .inner{
-        height:100%;
+        background-repeat: no-repeat;
+        background-position: center;
+        background-size: cover; 
+        height:auto;
+        line-height: auto;
+        margin:0 -2.5% 0 -2.5%;
         background: rgba(138, 134, 134, 0.575);
       }
 
       .title{
         padding:100px;
         margin-top:-30px;
+        text-align:center;
       }
       .center{text-align:center}
       .textBg{
         background:rgb(209, 206, 206);
-        height:100%;
-        width:40%;
-        /* text-align: center; */
-        margin-left: 30%;
-        margin-right:20%;
+        position:absolute;
+        left:15%;
+        right:15%;
+        bottom:5%;
+        top:10%;
+        text-align: center;
+
+        height:auto;
+        width:auto;
+        /* margin-left: auto;
+        margin-right: auto;
         margin-top:-5%;
-        margin-bottom:100px;
+        margin-bottom:10%; */
```

Comment:
why add */ here?

Reply:
I delete the codes from line 61 to line64. The */ is the end signal of the delete operation.

Reply:
I already removed it form the script